app/views/application/_flash_messages.html.erb

```diff
@@ -3,10 +3,19 @@
     <div class="row">
       <div class="col">
         <% flash.each do |type, msg| %>
+          <% if msg.kind_of?(Array) %>
```

Comment:
ło panie - sprawdzanie typu na widoku. To jest tak bardzo niespodziewana operacja jak niespodziewany jest passat ze szczerego złota w Wąchocku 😅. Coś ewidentnie robisz źle wyżej, ze tutaj potrzebujesz takich cudów

Reply:
Staram się unikać logiki na widokach jak tylko mogę i tutaj też długo szukałem lepszego rozwiazania. Ostatecznie przerzuciłem to do PayloadObject, update-ując metody które zwracają mi message i errory do kontrolera z serwisów. Teraz zwracam to w postaci stringa jako gotowe sentence. Myślałem też nad zagnieżdzeniem nawet pojedyczego stringa do arraya, a potem właśnie w tym miejscu iterować, ale już bez sprawdzania typu. Ale uznałem to trochę za code smell